include/SDL3/SDL_hints.h

```diff
@@ -2293,16 +2293,31 @@ extern "C" {
  * A variable controlling whether the hardware cursor stays visible when
  * relative mode is active.
  *
- * This variable can be set to the following values: "0" - The cursor will be
- * hidden while relative mode is active (default) "1" - The cursor will remain
- * visible while relative mode is active
+ * This variable can be set to the following values: 
+ * - "0": The cursor will be hidden while relative mode is active (default) 
+ * - "1": The cursor will remain visible while relative mode is active
  *
  * Note that for systems without raw hardware inputs, relative mode is
  * implemented using warping, so the hardware cursor will visibly warp between
  * frames if this is enabled on those systems.
  */
 #define SDL_HINT_MOUSE_RELATIVE_CURSOR_VISIBLE  "SDL_MOUSE_RELATIVE_CURSOR_VISIBLE"
 
+/**
+ * A variable controlling whether separate platform raw mouse events 
+ * are always generated whether or not relative mode is active.
+ *
+ * This variable can be set to the following values: 
+ * - "0": only generate platform raw mouse events while relative mode is active (default) 
+ * - "1": platform raw mouse events are always generated
+ *
+ * Note that raw mouse events are only available on some platforms, 
+ * they are provided independently of generic mouse events.
+ *
+ * \since This hint is available since SDL 3.0.0.
+ */
+#define SDL_HINT_MOUSE_RAW_EVENT_ALWAYS_ON  "SDL_MOUSE_RAW_EVENT_ALWAYS_ON"
```

Comment:
I probably would just make the event be disabled by default, and use the application enabling it as a trigger to enable/disable the motion. You can use `SDL_EventEnabled()` to check it.

Reply:
Call SDL_SetEventEnabled() in SDL_InitEvents()